corehq/apps/cloudcare/templates/formplayer/session_list.html

```diff
@@ -16,6 +16,12 @@
 </script>
 
 <script type="text/template" id="session-view-list-web-apps-template">
+    <div class="module-banner" style="background-color:#ffe3c2;">
```

Comment:
Inline style and hard-coded color values are bad news. How about using `.alert` and `.alert-info`?